### ISSUE TITLE
LeftProjection.get warning (Magnolia issue)

### ISSUE THREAD

**Issue by markus1189** [softwaremill/diffx#57]
We use `fatal-warnings` and using `diffx` produces the following warning:

```
[warn] /home/markus/repos/clones/diffx/core/src/test/scala/com/softwaremill/diffx/DiffTest.scala:375:22: method get in class LeftProjection is deprecated (since 2.13.0): use `Either.swap.getOrElse` instead
```

The issue is in magnolia: https://github.com/propensive/magnolia/pull/192

Once a new version is released, it would be cool to update this library as well :)

**Comment by markus1189** [comment]
The issue is fixed in magnolia and the current version used is new enough, are there any plans to have a new release :smile:

**Comment by ghostbuster91** [comment]
Released as 0.3.16 :tada:

**Comment by markus1189** [comment]
Awesome thanks!
